src/linters.ts

```diff
@@ -306,6 +306,10 @@ function lintBindingExpression(parsed,
                                implicitBindings,
                                name) {
   var undeclaredBindings = [];
+  properties = properties.concat([{
```

Comment:
This is the load bearing part of this change, the rest is just to fix the weirdness in the way this code was doing modules